### DISCUSSION TITLE
How to get speaker audio.

### DISCUSSION THREAD

**Discussion by Sillylittleguy1** [ading2210/shimboot#162] [Q&A]
I am on a pre built image for corsola, I have Bluetooth audio, but don't have any speaker audio by default. If anyone on any board has instructions on how you got it to work, it would be appreciated.

**Comment by nathblade16** [comment]
you prob can't

**Comment by Gaboom63** [comment]
@Sillylittleguy1 Can you make an issue using Documentation please, because speakers are untested. The 3rd Option is speakers. 
![image](https://github.com/user-attachments/assets/d0394b58-bff4-47f4-9597-11fc0550c4d6)
